python/fastquant/backtest/backtest.py

```diff
@@ -138,7 +138,7 @@ def backtest(
             cerebro.optstrategy(
                 STRATEGY_MAPPING[strat],
                 init_cash=[init_cash],
-                transaction_logging=[True if return_transactions else verbose],
```

Comment:
No longer needed since returning the history is now independent of `transaction_logging`